src/Components/Shared/ButtonForm/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './button.module.css';
+
+function ButtonForm({ nameImg, onAction }) {
+  return (
+    <div>
+      <img
+        className={styles.icon}
+        onClick={onAction}
+        src={`${process.env.PUBLIC_URL}/assets/images/${nameImg}`}
```

Comment:
you could use in the route of the image that is seen on the other side in case the route is not the same